README.rst

```diff
@@ -252,18 +255,18 @@ Install Lightning-GPU with MPI
     ``CUDA-aware MPI`` allows data exchange between GPU memory spaces of different nodes without the need for CPU-mediated transfers.
     Both the ``MPICH`` and ``OpenMPI`` libraries are supported, provided they are compiled with CUDA support.
     It is recommended to install the ``NVIDIA cuQuantum SDK`` and ``mpi4py`` Python package within ``pip`` or ``conda`` inside a virtual environment.
-    Please consult the `cuQuantum SDK`_ , `mpi4py <https://mpi4py.readthedocs.io/en/stable/install.html>`_,
+    Please consult the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ , `mpi4py <https://mpi4py.readthedocs.io/en/stable/install.html>`_,
     `MPICH <https://www.mpich.org/static/downloads/4.1.1/mpich-4.1.1-README.txt>`_, or `OpenMPI <https://www.open-mpi.org/faq/?category=buildcuda>`_ install guide for more information.
 
 **Before installing Lightning-GPU with MPI support using the direct SDK path, please ensure that:**
 
 .. note::
 
-    - **Lightning-Qubit**, ``CUDA-aware MPI`` **and** ``custatevec`` **are installed**.
-    - **The environment variable** ``CUQUANTUM_SDK`` **is set properly**.
-    - **Add the** ``path/to/libmpi.so`` **to** ``LD_LIBRARY_PATH``.
+    - Lightning-Qubit, ``CUDA-aware MPI`` and ``custatevec-cu12`` are installed.
+    - The environment variable ``CUQUANTUM_SDK`` is set properly.
```

Comment:
> The environment variable ``CUQUANTUM_SDK`` is set properly.

Does "properly" mean the path pointing to the cuquantum SDK location? If so, maybe we can say that so it's clear.

Reply:
yes, a bash command is given to set it in the following context:
`export CUQUANTUM_SDK=$(python -c "import site; print( f'{site.getsitepackages()[0]}/cuquantum')") `